internal/provider/resource_offer.go

```diff
@@ -2,173 +2,255 @@ package provider
 
 import (
 	"context"
+	"fmt"
 	"strings"
 
-	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
-	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	frameworkdiags "github.com/hashicorp/terraform-plugin-framework/diag"
```

Comment:
No need to rename this package.

Reply:
yeah I used it while developing (clashed with the sdk2) and forgot to get it back, probably should've renamed the other one :)